lib/components/form-fieldset.vue

```diff
@@ -27,12 +38,15 @@
                 return this.horizontal ? ('col-sm-' + (12 - this.labelSize)) : 'col-12';
             }
         },
-        mounted() {
-            const content = this.$refs.content;
-            if (!content) {
-                return;
+        methods: {
+            target() {
```

Comment:
The thing I'm sure $refs is only accessible​ after mounted() so we may keep it as a method but just call it inside them :)